common/store.ts

```diff
@@ -151,4 +155,23 @@ const configureStore = () => {
   return store;
 };
 
+function loadAndDedupeCustomTokens(network: NetworkConfig): CustomTokenState {
```

Comment:
Would be nice to see a unit test around this, if only to prevent future regressions. There was a fairly severe bug related to handling custom/default token conflicts, and ensuring we are covering all cases would be valuable.

Reply:
Ah, I see some tests in https://github.com/MyEtherWallet/MyEtherWallet/pull/768/files. Not sure if all cases are covered however.

Reply:
Those tests are only for the json files. I agree that it would be wise to make unit tests for this dedupe behavior though. I'll do a little refactor to make this take in both the network AND the local storage tokens, so that I can send it mocked lists.